### PR TITLE
py sol for usaco-1276

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/bronze/usaco-1276.mdx [hunk 1]
```diff
@@ -122,7 +122,77 @@ int main() {
...
+```py
+cows = []  # List to store {s, t, c} for each cow
+air_conditioners = []  # List to store {a, b, p, m} for each air conditioner
+min_cost = float(
+	"inf"
+)  # the minimum amount of money needed to keep all cows comfortable
+
+
+# Based on 'uses', determine if the current subset of air conditioners suffices
+# the constraints, and if so, update the minimum cost
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1276.mdx:136]: docstring

Comment (freakin23) [reply solutions/bronze/usaco-1276.mdx]: Done

File: solutions/bronze/usaco-1276.mdx [hunk 2]
```diff
@@ -131,9 +131,10 @@ min_cost = float(
 	"inf"
 )  # the minimum amount of money needed to keep all cows comfortable
 
-
-# Based on 'uses', determine if the current subset of air conditioners suffices
-# the constraints, and if so, update the minimum cost
+"""
+Based on 'uses', determine if the current subset of air conditioners suffices
+the constraints, and if so, update the minimum cost
+"""
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1276.mdx:137]: yk docstrings go below the function declaration right

Comment (freakin23) [reply solutions/bronze/usaco-1276.mdx:137]: I had no idea about it

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by freakin23** [comment]
Kevin, your previous commit was missing some double quotes, which is why the checks weren't completed. I had to do it, which eventually disabled the auto-merge.

**Pull request merged**